src/App.tsx

```diff
@@ -26,70 +27,139 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  if (sortType !== SortType.NONE) {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        visibleGoods.sort((firstGood, secondGood) => (
+          firstGood.localeCompare(secondGood)
+        ));
+        break;
+
+      case SortType.LENGTH:
+        visibleGoods.sort((firstGood, secondGood) => (
+          firstGood.length - secondGood.length
+        ));
+        break;
+
+      default:
+        break;
```

Comment:
```suggestion
			case SortType.NONE:
      default:
        break;
```

Reply:
![image](https://user-images.githubusercontent.com/116748103/210780018-438353d8-bf4e-456e-9b4b-f166752b5dc8.png)

Why would I add this case if all the sorting starts only if(sortType !== SortType.NONE)?

Reply:
I may be wrong, of course, but I'm just don't undestand

Reply:
@ivan-kadykalo 
Okay, done👍
(thx, for the explanation, btw)